File: Module/PlayWays/src/main/java-songmanage/com/module/playways/songmanager/view/ISongManageView.java
package com.module.playways.songmanager.view;

import com.module.playways.songmanager.model.RecommendTagModel;

import java.util.List;

public interface ISongManageView {
    void showRoomName(String roomName);

    void showRecommendSong(List<RecommendTagModel> recommendTagModelList);
}

File: Module/PlayWays/src/main/java-songmanage/com/module/playways/songmanager/view/ISongManageView.kt
package com.module.playways.songmanager.view

import com.module.playways.songmanager.model.RecommendTagModel

interface ISongManageView {
    fun showRoomName(roomName: String?)

    fun showRecommendSong(recommendTagModelList: MutableList<RecommendTagModel>)

    fun showAddSongCnt(cnt : Int)
}
